### DISCUSSION TITLE
Update of Google Play docs

### DISCUSSION THREAD

**Discussion by bschmalb** [codemagic-ci-cd/codemagic-docs#2209] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Setup Google Play publishing as described in these docs: https://docs.codemagic.io/flutter-publishing/publishing-to-google-play

### Expected results

The expected result is that the apps should be published in the Google Play Console.

### Actual results

```
Build completed successfully.
Following submissions will start shortly after App Store Connect completes processing of uploaded artifacts: "What to test" notes, App Store review.
Further logs are available in Post processing section.

Publishing app-prod-release.aab to Google Play
Published app-prod-release.aab to track production
Google Play failed to upload artefacts. The caller does not have permission: {
    "error": {
        "code": 403,
        "message": "The caller does not have permission",
        "status": "PERMISSION_DENIED"
    }
}

Build failed :|


Publishing failed :|
Google Play failed to upload artefacts. The caller does not have permission: {
    "error": {
        "code": 403,
        "message": "The caller does not have permission",
        "status": "PERMISSION_DENIED"
    }
}
```

#### Solution/Reason
I already found the reason for this problem, but he docs need to be updated as one step misguides the user in the correct setup and will quickly describe it here:

In **step 11** the [docs](https://docs.codemagic.io/flutter-publishing/publishing-to-google-play) read:

> On the **Account permissions** tab, leave everything as it is. (There is NO need to grant the service account **Admin** access).

It seems that this is not valid anymore, as the default permission do not allow the service account to _Release to production_.
The below seen permissions are **NOT** given by default as described in step 11 of the setup process and need to be **manually set**.

![Screenshot 2023-11-20 at 10 20 46](https://github.com/codemagic-ci-cd/codemagic-docs/assets/32933389/95667e3d-7dd0-46cc-bdd8-d1c4650d14d0)

The first permissions in the **Account permissions** is **needed** for the publishing to work. The below ones are **optional** if you want for the test tracks, testers, and store presence to be enabled as apparent.

### Build id (optional)

6557486a8d8a8d4c4b7d3d40

**Comment by himesh-cm** [comment]
Hi @bschmalb, thank you for pointing this out. We will review this internally and do the needful.

**Reply by himesh-cm** [reply]
Hi @bschmalb, We have raised an issue on https://github.com/codemagic-ci-cd/codemagic-docs/issues/2245 which you can track.
I will update this thread as soon as I can.

While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!

**Reply by himesh-cm** [reply]
Hi @bschmalb, these changes can be done in `app` level settings instead of `account` level. We have updated the docs as well.

**Comment by Mapreatt** [comment]
Hello developer am seeking for someone to  help me upload my game to their google play console 
I have issue with my account so i want to rent 
Please this is paid task i pay 20$ budget i pay to any payment method you want 
Please am seeking for help
